app/services/websocket_server.py

```diff
@@ -231,6 +231,36 @@ def format_order(record):
 
     await sio.emit('market', response, room=sid)
 
+#return ticker information, if no token specified show all
+@sio.on('returnTicker')
```

Comment:
I don't think there's a separate `returnTicker` event: instead, `returnTicker` is a key on a `market` event, which is emitted to a specific user in response to a `getMarket` event by that user.